src/node_options.h

```diff
@@ -126,7 +126,7 @@ class EnvironmentOptions : public Options {
   bool experimental_fetch = true;
   bool experimental_websocket = true;
   bool experimental_sqlite = true;
-  bool experimental_webstorage = false;
+  bool experimental_webstorage = true;
```

Comment:
If we're graduating webstorage to stable, then this should be renamed internally to just something like `bool webstorage = ...` or something